docs: Add explanations about fancy notifications. (#2)
wiris/thoth: add-in/fancy-notification/README.md

# Fancy notifications

Fancy notifications are a special way of communication with the MathType add-in for Microsoft Office users, mainly used to explain some concepts with the help of animated images.

## How to set a fancy notification

> You can set one, and only one, fancy notification per language. To do so, create (or modify it, if already present) a JSON file in this very folder with the format: `{language code}.json` where `language code` must be one of the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes.

Reviewer: ```suggestion
You can set one, and only one, fancy notification per language. To do so, create (or modify if already present) a JSON file in this very folder with the format: `{language code}.json` where `language code` must be one of the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes.
```

> This files must contain the following JSON structure:

Reviewer: ```suggestion
This file must contain the following JSON structure:
```

```json
{
  "id": [String, mandatory],
  "info": {
    "title": [String, optional]
  },
  "image": {
    "url": [URL, mandatory],
    "altText": [String, mandatory],
    "target": [URL, optional]
  }
}
```

## JSON fields explanation

* **id**: Mandatory string used to identify any given fancy notification. This identifier is necessary for the add-in internals, and must be different unique.
* **info.title**: Optional string. If present, will be used as the alert title.
* **image.url**(*): Mandatory URL. Points to the animated image (i.e.: the GIF file) to be shown.

> * **image.altText**: Mandatory string. Used as accesible text for the animated image.

Reviewer: ```suggestion
* **image.altText**: Mandatory string. Used as text to be spoken by screen readers when they come across the animated image.
```
